packages/@okta/vuepress-site/guides/sign-into-web-app/get-user-info/index.md

```diff
@@ -3,7 +3,7 @@ title: Get Info About the User
 ---
 When a user signs in, their profile information (stored in Okta) is made available to your application. It's common to use this info to update your application's UI.
 
-By default, the profile items (called "claims") that Okta returns include the user's email address, first name, and last name. 
+By default, the profile items (called "claims") that Okta returns include the user's email address, name and preferred username. 
```

Comment:
first and last name are not default claims (at least in the org I checked), assuming scopes of `email, profile, openid` only string based claims are `email` `name`, and `preferred_username`